src/data/languages/French.json

```diff
@@ -0,0 +1,575 @@
+"$language_Norwegian": "Norvégien",
+"$default_filename": "Fichier d'Anilist.co",
+"$generic_anime": "Anime",
+"$generic_manga": "Manga",
+"$page": "Page {0}",
+"$dubMarker_notice": "{0} doublage(s) existe(nt)",
+"$button_submit": "Envoyer",
+"$button_search": "Rechercher",
+"$button_run": "Executer",
+"$button_add": "Ajouter",
+"$button_reset": "Réinitialiser",
+"$button_resetAll": "Tout réinitialiser",
+"$button_defaultSettings": "Paramètres par défaut",
+"$missing_N/A_data": "-",
+"$button_next": "Suivant →",
+"$button_previous": "← Précédent",
+"$button_refresh": "Rafraichir",
+"$button_edit": "Modifier",
+"$button_publish": "Publier",
+"$button_cancel": "Annuler",
+"$button_review": "Écrire une critique",
+"$placeholder_status": "Écrire un status...",
+"$placeholder_reply": "Écrire une réponse...",
+"$placeholder_message": "Écrire un message...",
+"$placeholder_forum": "Écrire un message dans le forum...",
+"$forumMedia_backlink": "Add a link back to a work's database page on its forum feed",
+"$settings_title": "Paramètres d'Automail",
+"$profile_title": "Profil de {0}",
+"$notImplemented": "Cette fonctionnalité n'est pas encore implémentée",
+"$settings_version": "Version: ",
+"$settings_homepage": "Page d'accueil: ",
+"$settings_repository": "Répertoire: ",
+"$settings_moreInfo_tooltip": "Plus d'informations",
+"$settings_category_Notifications": "Notifications",
+"$settings_category_Feeds": "Flux",
+"$settings_category_Forum": "Forum",
+"$settings_category_Lists": "Listes",
+"$settings_category_Profiles": "Profiles",
+"$settings_category_Stats": "Stats",
+"$settings_category_Media": "Média",
+"$settings_category_Navigation": "Navigation",
+"$settings_category_Browse": "Feuilleter",
+"$settings_category_Script": "Scripte",
+"$settings_category_Login": "Connexion",
+"$settings_category_Newly Added": "Ajouts récents",
+"$settings_button_export": "Exporter les paramètres",
+"$settings_export_description": "Il peut être utile de conserver une copie en cas de supression du cache ou du stockage du navigateur, ce qui entrainerait la perte de vos paramètres Automail.",
+"$settings_import": "Importer des paramètres",
+"$settings_experimental_suffix": "[EXPÉRIMENTAL]",
+"$settings_partialLocalisationLanguage_description": "Langue d'Automail",
+"$settings_CSSadd": "Ajouter du CSS personalisé à votre profil. Celui-ci sera visible aux autres utilisateurs d'Automail.",
+"$settings_CSSlinkTip": "(Vous pouvez utiliser un lien direct vers une feuille de style)",
+"$settings_pinnedActivity": "Ajouter une activité épinglée au profil",
+"$settings_notificationDotColour": "Couleurs des points de notification",
+"$setting_notifications": "Amélioration des notifications",
+"$setting_moreStats": "Afficher un onglet supplémentaire aux statistiques",
+"$setting_compare": "Remplacer la fonctionnalité de comparaison d'origine",
+"$setting_CSSsmileyScore": "Donner aux smileys des couleurs distinctes",
+"$setting_tweets": "Embed linked tweets",
```

Comment:
It's a bit easier to maintain a translation if untranslated keys are left out. The script uses the fallback language then, which has the up to date version. Missing keys are easy to find later with the /src/data/languages/diff_tool.html tool.